src/util-memcmp.c

```diff
@@ -188,8 +161,12 @@ static int MemcmpTest14 (void)
 #ifdef PROFILING
     uint64_t ticks_start = 0;
     uint64_t ticks_end = 0;
-    const char *a[] = { "0123456789012345", "abc", "abcdefghij", "suricata", "test", "xyz", "rrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrr", "abcdefghijklmnopqrstuvwxyz", NULL };
-    const char *b[] = { "1234567890123456", "abc", "abcdefghik", "suricatb", "test", "xyz", "rrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrr", "abcdefghijklmnopqrstuvwxyz", NULL };
+    const char *a[] = { "0123456789012345", "abc", "abcdefghij", "suricata", "test", "xyz",
+        "rrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrr",
+        "abcdefghijklmnopqrstuvwxyz", NULL };
+    const char *b[] = { "1234567890123456", "abc", "abcdefghik", "suricatb", "test", "xyz",
+        "rrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrr",
+        "abcdefghijklmnopqrstuvwxyz", NULL };
```

Comment:
Please remove these from the commit. This commit must only have the FAIL/PASS API changes, no other changes should be a part of it.

Reply:
Okay, noted